protobuf/concept.proto

```diff
@@ -286,72 +222,67 @@ message Thing {
     }
 
     message GetHas {
-        message Iter {
-            message Req {
-                repeated Type attributeTypes = 1;
-                bool keysOnly = 2;
-            }
-            message Res {
-                Thing attribute = 1;
-            }
+        message Req {
+            repeated Type attribute_types = 1;
+            bool keys_only = 2;
+        }
+        // @Iterable
+        message Res {
+            Thing attribute = 1;
```

Comment:
Having an intermediate message name `Iter` seemed unnecessarily verbose. We now flag iterable messages using comments.